### PR TITLE
Meaning of the SPDX acronym has changed in SPDX 3.0

### PR DESCRIPTION
**Pull request by vargenau**

### CODE REVIEW

File: docs/annexes/diffs-from-previous-editions.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 # A.1 Differences between V3.0 and V2.3 <a name="A.1"></a>
 
+## SPDX meaning
+
+In previous editions of the specification, SPDX meant "Software Package Data Exchange".
+
+Starting with V3.0, SPDX means "System Package Data Exchange".
```
Comment (goneall) [inline docs/annexes/diffs-from-previous-editions.md:9]:
```suggestion
Starting with V3.0, the scope of SPDX has expanded beyond software and now means "System Package Data Exchange".
```

**Review by goneall — CHANGES_REQUESTED** [review]
Suggestion to add a bit of context

**Comment by bact** [comment]
Can we also add this to the Glossary as well?

**Comment by vargenau** [comment]
> Can we also add this to the Glossary as well?

I have created https://github.com/spdx/spdx-3-model/pull/801 for that.

**Review by kestewart — APPROVED** [review]
Thanks!  Looks good.

**Review by goneall — APPROVED** [review]
Resolved merge conflicts and changes look good

**Comment by goneall** [comment]
Replaced with PR #1030 - could not resolve merge conflicts with this PR

**Pull request closed without merging**